scripts/prepare_heat_demand.py

```diff
@@ -10,6 +10,9 @@
 in_path3 : str
     ``raw/holidays.csv``: path of input file with holidays of all states in Germany as .csv
 in_path4 : str
+    ``raw/building_class.csv``: path of input file with building classes of all states in Germany
+    as .csv
+in_path5 : str
```

Comment:
Seems like you have nr. 5 twice. Please make sure that this documentation corresponds with the current inputs of the script.

Reply:
Oh right, thank you. Done with commit https://github.com/rl-institut/oemof-B3/pull/76/commits/a9c3cb64d8ac5304295be9e57ea36503e687742d.